### PR TITLE
Footer created

### PR DESCRIPTION
**Pull request by Erikisak**
footer created for both mobile and desktop
responsive to all screens
created after figma-design 
Closes issue #26

### CODE REVIEW

File: src/Components/Footer.tsx [hunk 1]
```diff
@@ -1,9 +1,134 @@
...
+        <FacebookIcon sx={iconStyle}/>
+      </Box>
+
+
+      </Box>
+
+
+
+
+
```
Comment (liinneea-a) [inline src/Components/Footer.tsx:64]: You have to fix these whitespaces

File: src/Components/Footer.tsx [hunk 2]
```diff
@@ -1,9 +1,134 @@
...
+      <Box sx={iconAlignDesktop}>
+        <TwitterIcon sx={iconStyle} />
+        <InstagramIcon sx={iconStyle}/>
+        <FacebookIcon sx={iconStyle}/>
+      </Box>
+
+
+      </Box>
+
+
```
Comment (mirrenil) [inline src/Components/Footer.tsx:61]: Can you remove all the white-space?

**Review by mirrenil — CHANGES_REQUESTED** [review]
Skriv en beskrivning av PRen, finns riktlinjer i readme. Och formatera så att white-space försvinner 👍

File: src/Components/Footer.tsx [hunk 3]
```diff
@@ -1,9 +1,134 @@
...
+      </Box>
+
+      
+      
   )
 }
 
+const navBox: SxProps = {
+}
+const test: SxProps = {
```
Comment (liinneea-a) [inline src/Components/Footer.tsx:74]: I have no idea what element or section this is styles. Please rename this

File: src/Components/Footer.tsx [hunk 4]
```diff
@@ -1,9 +1,134 @@
...
+  position: 'relative',
+  display: { xs: 'none', md:'flex', lg: 'flex', xl: 'flex' },
+  
+}
+const textAlign: SxProps = {
+  marginBottom: '30px'
+}
+const align1: SxProps = {
```
Comment (liinneea-a) [inline src/Components/Footer.tsx:82]: Why is this namned align when the properties says something else?

File: src/Components/Footer.tsx [hunk 5]
```diff
@@ -1,9 +1,134 @@
...
+  display: { xs: 'flex', md:'none', lg: 'none', xl: 'none' },
+  flexDirection: 'column',
+  paddingBottom: '20px',
+}
+const align2: SxProps = {
```
Comment (liinneea-a) [inline src/Components/Footer.tsx:87]: Same for this! :)

File: src/Components/Footer.tsx [hunk 6]
```diff
@@ -1,9 +1,134 @@
...
+  display: { xs: 'none', md:'flex', lg: 'flex', xl: 'flex' },
+  flexDirection: 'column',
+  position: 'absolute',
+  bottom: { xs: 'none', md:'0', lg: '10px', xl: '30px' },
+  left: '60px',
+  fontSize: { xs: 'none', md:'20px', lg: '30px', xl: '30px' },
+  boxSizing: 'border-box'
+}
+const align: SxProps = {
```
Comment (liinneea-a) [inline src/Components/Footer.tsx:96]: And this!

File: src/Components/Footer.tsx [hunk 7]
```diff
@@ -1,9 +1,134 @@
...
+  flexDirection: 'row',
+  position: 'absolute',
+  bottom: '30px',
+  right: { xs: 'none', md:'200px', lg: '200px', xl: '300px' },
+}
+const iconStyle: SxProps = {
+  fontSize: { xs: 'none', md:'60px', lg: '70px', xl: '70px' },
+}
+const textalign: SxProps = {
+
```
Comment (liinneea-a) [inline src/Components/Footer.tsx:132]: Remove this empty rule set

File: src/Components/Footer.tsx [hunk 8]
```diff
@@ -1,9 +1,134 @@
...
+}
+const align: SxProps = {
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  gap: '10px',
+  marginLeft: '50px',
+  
+
+
```
Comment (liinneea-a) [inline src/Components/Footer.tsx:104]: Remove unnecessary whitespaces :)

**Review by liinneea-a — CHANGES_REQUESTED** [review]
Good job! But you have to remove whitespaces and rename a few things before it gets merged :)

**Review by liinneea-a — APPROVED** [review]

**Review by mirrenil — APPROVED** [review]

**Pull request merged**